app/models/zone.rb

```diff
@@ -19,6 +20,8 @@ class Zone < ApplicationRecord
   has_many :vms,                 :through => :ext_management_systems
   has_many :miq_templates,       :through => :ext_management_systems
   has_many :ems_clusters,        :through => :ext_management_systems
+  has_many :container_nodes,     :through => :container_managers
+  has_many :container_projects,  :through => :container_managers
```

Comment:
what other container entities make sense here? Pods? Containers?

Reply:
I'm not sure anything except EMS and N=ode is possible in UI.